### DISCUSSION TITLE
Any chance of exposing siren but not camera feed?

### DISCUSSION THREAD

**Discussion by slawnz** [dgreif/ring#1101] [General]
Hi, I would love the ability to have the siren switches for my cameras in Homekit without the camera feeds themselves being there. Reason is I share my 'home' with my kids, and I don't want them to be able to view the cameras. If I disable the cameras, I can no longer trigger the sirens in automations. Is there any possibility of this option being added to the plugin setup? Thanks!

**Comment by dgreif** [comment]
Sorry, that level of customization isn't currently supported and isn't something I'd like to add/maintain
